show-vars.md

```diff
@@ -0,0 +1,121 @@
+---
+title: SHOW (session settings)
+summary: The SHOW statement displays the current settings for the client session.
+toc: false
+---
+
+The `SHOW` [statement](sql-statements.html) can display the value of either one or all of
+the session setting variables. Some of these can also be configured via [`SET`](set-vars.html).
+
+<div id="toc"></div>
+
+## Required Privileges
+
+No [privileges](privileges.html) are required to display the session settings.
+
+## Synopsis
+
+{% include sql/diagrams/show_var.html %}
+{% include sql/diagrams/show_all.html %}
+
+The variable name is case insensitive.
+It may be enclosed in double quotes; this is useful if the variable name itself contains spaces.
+
+Special syntax cases supported for compatibility:
+
+| Syntax | Equivalent to |
+|--------|---------------|
+| `SHOW TRANSACTION PRIORITY` | `SHOW "transaction priority" |
+| `SHOW TRANSACTION ISOLATION LEVEL` | `SHOW "transaction isolation level" |
+| `SHOW TIME ZONE` | `SHOW "time zone" |
+| `SHOW TRANSACTION STATUS` | `SHOW "transaction status" |
+
+{{site.data.alerts.callout_info}}
```

Comment:
Unfortunately, this alert needs to be formatted as HTML. As is, it's breaking the table above it. To save time, here's the HTM version for you to use:

```
{{site.data.alerts.callout_info}}The <code>SHOW</code> statement for session settings is unrelated to the other <code>SHOW</code> statements: <a href="show-create-table.html"><code>SHOW CREATE TABLE</code></a>, <a href="show-create-view.html"><code>SHOW CREATE VIEW</code></a>, <a href="show-users.html"><code>SHOW USERS</code></a>, <a href="show-databases.html"><code>SHOW DATABASES</code></a>, <a href="show-columns.html"><code>SHOW COLUMNS</code></a>, <a href="show-grants.html"><code>SHOW GRANTS</code></a>, and <a href="show-constraints.html"><code>SHOW CONSTRAINTS</code></a>.
```